blog/2025-02-21/index.md

```diff
@@ -104,15 +141,22 @@ Once an app is accepted onto Flathub, it’s not just the wild west; there are s
 - **Static permission changes are held for human review** before an update is released to users
 - **Critical MetaInfo changes are held for human review**, e.g. if an app name, developer name, app summary, or license changes
 
+![Build moderation email for Kodi showing permission changes, light](moderation-light.png#gh-light-mode-only)
```

Comment:
Technically, the used sceenshots now are not of the email - it's the developer side of the website

Reply:
changed